agir/events/models.py

```diff
@@ -1048,3 +1048,60 @@ def __str__(self):
 
     class Meta:
         verbose_name = "Visio-conférence"
+
+
+class Invitation(TimeStampedModel):
+
+    INVITATION_PENDING = "pending"
+    INVITATION_ACCEPTED = "accepted"
+    INVITATION_REFUSED = "refused"
+
+    STATUSES = (
+        (INVITATION_PENDING, "En attente"),
+        (INVITATION_ACCEPTED, "Acceptée"),
+        (INVITATION_REFUSED, "Refusée"),
+    )
```

Comment:
Pas besoin d'`INVITATION` vu que ça donnera déjà `Invitation.STATUS_PENDING` et j'ai l'impression que c'est un peu une habitude dans le projet de nommer les constante NOM-DE-LA-PROP_VALUE (ex. Membership.TYPE_MEMBER)
```suggestion
    STATUS_PENDING = "pending"
    STATUS_ACCEPTED = "accepted"
    STATUS_REFUSED = "refused"

    STATUSES = (
        (STATUS_PENDING, "En attente"),
        (STATUS_ACCEPTED, "Acceptée"),
        (STATUS_REFUSED, "Refusée"),
    )
```